docs/rules/no-new-buffer.md

```diff
@@ -0,0 +1,27 @@
+# Enforce the use of `Buffer.from()` and `Buffer.alloc()` instead of the deprecated `new Buffer()`
+
+Enforces the use of [Buffer.from](https://nodejs.org/api/buffer.html#buffer_class_method_buffer_from_array) and [Buffer.alloc()](https://nodejs.org/api/buffer.html#buffer_class_method_buffer_alloc_size_fill_encoding) instead of [new Buffer()](https://nodejs.org/api/buffer.html#buffer_new_buffer_array) which has been deprecated.
```

Comment:
With the new sentence `which has been deprecated since Node.js 4`, I believe a comma makes sense.